src/nls/root/strings.js

```diff
@@ -398,6 +398,7 @@ define({
     "CMD_WORKINGSET_SORT_BY_TYPE"         : "Sort by Type",
     "CMD_WORKING_SORT_TOGGLE_AUTO"        : "Automatic Sort",
     "CMD_THEMES"                          : "Themes\u2026",
+    "CMD_TOGGLE_SEARCH_AUTOHIDE"          : "Automatically close search",
```

Comment:
Should most likely be consistent with #L763 (`DESCRIPTION_SEARCH_AUTOHIDE`) (Hide/close, search / search bar)